src/main.rs

```diff
@@ -65,8 +74,29 @@ fn main() {
     gtk::init().unwrap();
     env_logger::init();
 
-    let res = gio::Resource::load(config::RESOURCES_FILE).expect("Could not load gresource file");
-    gio::resources_register(&res);
+    let resources = match env::var("MESON_DEVENV") {
+        Err(_) => gio::Resource::load(config::RESOURCES_FILE.to_owned())
+            .expect("Unable to load resources.gresource"),
+        Ok(_) => match env::current_exe() {
+            Ok(path) => {
+                let mut resource_path = path;
+
+                for _ in 0..2 {
+                    resource_path.pop();
+                }
+                resource_path.push("share/geopard/resources.gresource");
+
+                gio::Resource::load(&resource_path)
+                    .expect("Unable to load resources.gresource in devenv")
+            }
+            Err(err) => {
+                error!("Unable to find the current path: {}", err);
+                process::exit(0x0100);
```

Comment:
In the documentation of `process::exit`:  "On most Unix-like platforms, only the eight least-significant bits are considered."
https://doc.rust-lang.org/std/process/fn.exit.html#platform-specific-behavior

Exiting with `0x0100` will lead to inconsistencies between platforms.